src/include/sof/audio/codec_adapter/codec/cadence.h

```diff
@@ -27,6 +27,7 @@ extern xa_codec_func_t xa_dabplus_dec;
 extern xa_codec_func_t xa_drm_dec;
 extern xa_codec_func_t xa_mp3_dec;
 extern xa_codec_func_t xa_sbc_dec;
+extern xa_codec_func_t xa_vorbis_dec;
```

Comment:
TODO verify if this is actually needed, given that the third party header is present.

Reply:
I think we still need to build this without Cadence library and use the pass through as the dummy Cadence library ? SO IIRC, this iw why we have the externs declared here ?